ethereum/slogging.py

```diff
@@ -1,9 +1,9 @@
+import sys
 import logging
 import json
 import textwrap
 from json.encoder import JSONEncoder
 from logging import StreamHandler, Formatter, FileHandler
-from ethereum.utils import bcolors, isnumeric
```

Comment:
@konradkonrad removing these two imports makes this module independent from the rest of the package.